timeline/tests/test_views_ajax.py

```diff
@@ -254,3 +273,69 @@ def test_get(self):
         }
         self.assertIn(expected['app'], str(response.data))
         self.assertIn(expected['user'], str(response.data))
+
+
+class TestEventStatusExtraAjaxView(TestEventAjaxViewsBase, EventExtraDataMixin):
+    """Tests for EventStatusExtraAjaxView"""
+
+    def setUp(self):
+        super().setUp()
+        self.event = self.make_event(
+            self.project, 'projectroles', self.user, 'test_event'
+        )
+        self.event_status_init = self.event.set_status(
+            'INIT',
+            DEFAULT_MESSAGES['INIT'],
+            extra_data={'test': 'test'},
+        )
+        self.event_site = self.make_event(
+            None, 'projectroles', self.user, 'test_event_site'
+        )
+        self.event_site_status_init = self.event_site.set_status(
+            'INIT',
+            DEFAULT_MESSAGES['INIT'],
+            extra_data={'test': 'test'},
+        )
+
+    def test_get(self):
+        """Test event status extra data retrieval"""
+        self.maxDiff = None
```

Comment:
Please clean up this and other manual `maxDiff` settings from tests, they're ok for debugging but let's not commit them :)